Tests/TestUtilities/TestLogger.cs

```diff
@@ -53,22 +53,26 @@ public void AssertWarningsLogged(int expectedCount)
         {
             Assert.AreEqual(expectedCount, this.Warnings.Count, "Unexpected number of warnings logged");
         }
+        public void AssertMessagesLogged(int expectedCount)
+        {
+            Assert.AreEqual(expectedCount, this.Messages.Count, "Unexpected number of messages logged");
+        }
 
         public void AssertMessageLogged(string expected)
         {
-            bool found = this.Messages.Any(s => expected.Equals(s, System.StringComparison.InvariantCulture));
+            bool found = this.Messages.Any(s => expected.Equals(s, System.StringComparison.CurrentCulture));
```

Comment:
@duncanpMS what is the purpose of this change? code that depends on the locale is usually a bad idea IMO